### PR TITLE
Adding files sharing

### PR DESCRIPTION
**Pull request by axeloz**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
Files Sharing |   * [Files Sharing](https://github.com/axeloz/filesharing) - Open Source and self-hosted files sharing application based on unique and temporary links..  `GPL-3.0` `PHP` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by DJCrashdummy** [comment]
my guess(es):
- [ ] 2 full stops at the end don't make sense at all
- [ ] the license is written different in the [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses)

File: README.md [hunk 1]
```diff
@@ -580,6 +580,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 
 #### File transfer/synchronization
 
+  * [Files Sharing](https://github.com/axeloz/filesharing) - Open Source and self-hosted files sharing application based on unique and temporary links..  `GPL-3.0` `PHP`
```
Comment (nodiscc) [inline README.md]: * Please move it to https://github.com/Kickball/awesome-selfhosted#single-clickdrag-n-drop-upload
 * License: `GPLv3`
 * Please remove the double dot and double space at the end of the description

Comment (axeloz) [inline README.md]: OK, will fix. FYI I have been using the Makefile so I didn't check the syntax at all, just copied and pasted. Thanks

**Comment by Kickball** [comment]
@nodiscc do we need to review the makefile? It seemed to generate the wrong syntax based on @axeloz's comment.

**Comment by axeloz** [comment]
@Kickball I'm not 100% sure that the Makefile was in fault. I should run another test to be sure.
Thanks anyways

**Comment by nodiscc** [comment]
@Kickball https://github.com/Kickball/awesome-selfhosted/pull/1128, I've clarified usage of `make add`

**Pull request merged**